Updating App Insights documentation

MichaelMendix opened the pull request:
Updated name "Product Insights" to "App Insights"
Remove deprecated settings instructions for feature we don't use anymore 
Updated screenshot to reflect removal of deprecated settings feature

MichaelMendix committed:
Add files via upload

MichaelMendix committed:
Update product-insights.md

changes Product Insights to App Insights
remove deprecated instructions for Settings tab
update screenshot

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/mendix/docs?pullRequest=5792) <br/>All committers have signed the CLA.

Adam-Dupaski commented:
@Luyao-Zhang-1 are you able to pick this up? I verified that it can be published immediately.

Luyao-Zhang-1 committed:
Use bold style for UX text

Luyao-Zhang-1 commented on content/en/docs/developerportal/collaborate/product-insights.md:
```diff
@@ -118,14 +118,6 @@ You can collect test data using two methods:
-{{< figure src="/attachments/developerportal/collaborate/product-insights/settings-icon.png" >}}
-
-Here you can view the existing API key, or generate an API key if there is not one available. You need to use the API key as input when you configure the App Insights – Mini-Surveys module in Studio Pro to implement the mini-survey. You need only one API key per app.
-
```
Then how can we get to **Product Insights Settings** to view/generate an API key? @MichaelMendix

MichaelMendix replied:
Ahhh, 😲 good question, thank you for finding this

So we moved the API Key Generate to the Survey Settings Tab Page (2.1.3 Settings Tab https://docs.mendix.com/developerportal/collaborate/product-insights/#settings)

I think API Key Generate has to be mentioned in  (2.1.3 Settings Tab https://docs.mendix.com/developerportal/collaborate/product-insights/#settings)

MichaelMendix replied:
@Luyao-Zhang-1 

Okay I have adjusted the documentation and also added a few more details/sections 😊

MichaelMendix committed:
Update product-insights.md

Updated settings page instructions for API Key Generation 
Added explanation for the difference between TestModes for all users & project members 
Added FAQ Section 
Added  Survey Pop-Up Behaviour

Luyao-Zhang-1 committed:
Delete settings-icon.png

Luyao-Zhang-1 committed:
Edit input text

Luyao-Zhang-1 committed:
Add "the"
